chromadb/test/property/test_add.py

```diff
@@ -14,13 +12,17 @@ def api(request):
     return chromadb.Client(configuration)
 
 
-@given(collection=strategies.collections(), embeddings=strategies.embeddings())
+@given(collection=strategies.collections(), embeddings=strategies.embedding_set())
 def test_add(api, collection, embeddings):
 
     api.reset()
 
-    coll = api.create_collection(**collection)
+    coll = api.create_collection(**collection, embedding_function=lambda x: None)
```

Comment:
Just leaving a TODO that across all of our new tests EF are relatively untested.